### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**
Had to change qt 5.9 install path since it's no longer the latest release.

### CODE REVIEW

File: depends/packages/qt.mk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PACKAGE=qt
 $(package)_version=5.9.7
-$(package)_download_path=https://download.qt.io/official_releases/qt/5.9/$($(package)_version)/submodules
```
Comment (xansec) [inline depends/packages/qt.mk:3]: Can you comment why this change was needed?

**Comment by rnshah9** [comment]
Qt has moved on to version 6, so they have changed the URL of version 5.9, which this repository uses. [Patched qt.mk](https://github.com/mayhemheroes/blocknet/pull/1/files/a3ea7484d9ec7d8568b006fbfb70211c95e92b82#diff-0d7e256978e78897b774581dad22102138ce4ba46aa07faaef1ae6fc6178aad4) to fix this.

**Comment by xansec** [comment]
LGTM
